rcl/include/rcl/graph.h

```diff
@@ -401,15 +401,15 @@ RCL_WARN_UNUSED
 rcl_ret_t
 rcl_names_and_types_fini(rcl_names_and_types_t * names_and_types);
 
-/// Return a list of available nodes in the ROS graph.
+/// Return a list of available node names in the ROS graph and a list of their namespaces.
```

Comment:
I think this might be a little clearer as:

```suggestion
/// Return a list of node names and their associated namespaces in the ROS graph.
```

Reply:
This is shows much clearer the relation between the two lists, thank you.